test/run_mmodel_tests.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
```

Comment:
Consider something like
```
set -euo pipefail
```
to avoid surprisingly silent failures. See e.g. http://redsymbol.net/articles/unofficial-bash-strict-mode/